Update object-not-appear-in-word-2010.md (#1571)
MicrosoftDocs/OfficeDocs-Support: Office/Client/word/object-not-appear-in-word-2010.md

3. Under **Printing options**, click to select the **Print drawings created in Word ** check box.   
4. Click **OK**.   

> 
> Another workaround, find the images that are no printing, **Select**. **Copy** and **Paste** them in a new word document. Now, **Select**, **Copy** and **Paste** them back in the oringal document after manually deleting the original ones. 

Reviewer: ```suggestion
Another workaround is to find the images that are no printing. Select them and copy and paste them into a new word document. Then, select them again and copy and paste them back into the original document after manually deleting the originals.
```
